bundles/org.openhab.binding.freeboxos/src/main/java/org/openhab/binding/freeboxos/internal/api/rest/FreeboxOsSession.java

```diff
@@ -75,7 +77,8 @@ public String initialize(FreeboxOsConfiguration configuration) throws FreeboxExc
     private String initiateConnection() throws FreeboxException {
         try {
             String localToken = appToken;
-            if (localToken != null) {
+            if (localToken != null && !localToken.isBlank()) {
+                session = null;
```

Comment:
Do not send session token to get challenge, if session is valid FreeboxOS API will not return challenge.